tornettools/plot.py

```diff
@@ -236,8 +242,13 @@ def __plot_transfer_time(args, circuittype, torperf_dbs, tornet_dbs, bytes_key):
     # cache the corresponding data in the 'data' keyword for __plot_cdf_figure
     for tornet_db in tornet_dbs:
         tornet_db['data'] = [tornet_db['dataset'][i][bytes_key] for i, _ in enumerate(tornet_db['dataset']) if bytes_key in tornet_db['dataset'][i]]
+    # We do not have public Tor data about 10MiB downloads but we are still
+    # interested in calculatin the transfer times for the simulated network
     for torperf_db in torperf_dbs:
-        torperf_db['data'] = [torperf_db['dataset']['download_times'][bytes_key]]
+        if int(bytes_key) <= 5242880:
```

Comment:
Ok for a one-off, but if the aim was to merge this we'd probably want to avoid hard coding this constant here, and instead just make this generally handle a size missing from the public data.

Reply:
I thought the idea was not to merge this like this. Should I refactor this so that it can be merged?

Reply:
No, it's fine do this as a one-off, was just double checking.

It'd be helpful to add a note to the PR description that this PR is for discussion purposes and isn't actually a request to merge